Source/Estimation/YINUtil.swift

```diff
@@ -0,0 +1,247 @@
+//
+//  YINUtil.swift
+//  Beethoven
+//
+//  Created by Guillaume Laurent on 09/10/16.
+//  Adapted from https://code.soundsoftware.ac.uk/projects/pyin/repository
+//  by Matthias Mauch, Centre for Digital Music, Queen Mary, University of London.
+//
+//
+
+import UIKit
+import Accelerate
+
+class YINUtil {
+
+  // slow and eats a lot of CPU, but working
+  //
+  class func difference2(buffer:[Float]) -> [Float] {
```

Comment:
@glaurent Is there any reason of having `class` functions here? Or if it's meant to be a set of functions, could be a struct with static `func`, right?